code/game/objects/items.dm

```diff
@@ -488,6 +488,16 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 /obj/item/proc/on_found(mob/finder)
 	return
 
+/**
+ * To be overwritten to only perform visual tasks;
+ * this is directly called instead of equipped on visual-only features like human dummies equipping outfits.
```

Comment:
```suggestion
 * this is directly called instead of `equipped` on visual-only features like human dummies equipping outfits.
```

Makes it clear this is a proc name and not just a word.